internal/controller/management.go

```diff
@@ -24,8 +24,8 @@ func NewManagementServer(cm *ConnectionManager, r *mux.Router) *ManagementServer
 }
 
 func (s *ManagementServer) Routes() {
-	s.router.HandleFunc("/connection/disconnect", s.handleDisconnect())
-	s.router.HandleFunc("/connection/status", s.handleConnectionStatus())
+	s.router.HandleFunc("/wss/receptor-controller/connection/disconnect", s.handleDisconnect())
```

Comment:
At this point, the /connection endpoint doesn't need to be public or related to the websocket connection.  I think we could leave the connection endpoint as /connection or /receptor-controller/connection.

Reply:
:+1: